app/main.py

```diff
@@ -1,6 +1,25 @@
+import requests
+import os
+
+
+weather_site = "https://api.weatherapi.com/v1/current.json?key="
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    city = "Paris"
+    api_key = os.environ["weatherapi_key"]
+    url = f"{weather_site}{api_key}&q={city}&aqi=no"
+
+    data = requests.get(url, params={"city": city})
```

Comment:
If you use argument  `params={...}` in request. Please don't write these parameters in the URL. 
`params={"key": API_KEY, "q": CITY}`